src/mlpack/tests/split_data_test.cpp

```diff
@@ -175,6 +175,20 @@ TEST_CASE("SplitLabeledDataResultMat", "[SplitDataTest]")
   CheckDuplication(std::get<2>(value), std::get<3>(value));
 }
 
+TEST_CASE("SplitCheckSize", "[SplitDataTest]")
+{
+  arma::mat input(2,10);
+  input.randu();
+
+  const Row<size_t> firstLabels = arma::linspace<Row<size_t>> (0, input.n_cols-1, input.n_cols);
+
+  const Row<size_t> secondLabels = arma::linspace<Row<size_t>> (0, input.n_cols, input.n_cols+1);
```

Comment:
```suggestion
  const Row<size_t> firstLabels = arma::linspace<Row<size_t>>(0, input.n_cols - 1, input.n_cols);

  const Row<size_t> secondLabels = arma::linspace<Row<size_t>>(0, input.n_cols, input.n_cols + 1);
```

Space between operators.